src/styles/index.scss

```diff
@@ -1,3 +1,32 @@
+@use './utils/variables' as v;
+@use './utils/mixins' as m;
+@import './utils/stars-block';
+@import './utils/card-block';
+@import './utils/container';
+@import './utils/header';
```

Comment:
Consider replacing `@import` with `@use` for these files to maintain consistency and take advantage of SCSS's module system. This will help avoid redundancy and potential conflicts.